### PR TITLE
added cache-path command

### PR DESCRIPTION
**Pull request by ametel01**
solves #558

### CODE REVIEW

File: scarb/src/bin/scarb/args.rs [hunk 1]
```diff
@@ -134,6 +136,8 @@ pub enum Command {
     Cache(CacheSubcommand),
     /// Remove generated artifacts.
     Clean,
+    /// Print cache path.
+    CachePath,
```
Comment (maciektr) [inline scarb/src/bin/scarb/args.rs:140]: Hey! `scarb cache clean` seems enough, I do not think we need an alias for that here.

File: scarb/src/bin/scarb/commands/cache_path.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+use anyhow::Result;
+use scarb::core::Config;
+
+use scarb::ops;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    ops::cache_path(config)
```
Comment (maciektr) [inline scarb/src/bin/scarb/commands/cache_path.rs:8]: Similarly to https://github.com/software-mansion/scarb/blob/main/scarb/src/bin/scarb/commands/manifest_path.rs, I do not think we need to move this to `ops`.

File: scarb/src/ops/cachepath.rs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+use anyhow::Result;
+
+use crate::core::Config;
+
+#[tracing::instrument(skip_all, level = "debug")]
+pub fn cache_path(config: &Config) -> Result<()> {
+    let path = config.dirs().cache_dir.path_unchecked();
+    if path.exists() {
```
Comment (maciektr) [inline scarb/src/ops/cachepath.rs:8]: I think we should output the path even if it is not existing in the filesystem yet.

**Review by maciektr — COMMENTED** [review]
Would you mind adding a test case for this?

File: scarb/src/bin/scarb/args.rs (hunk 1 above)
Comment (ametel01) [reply scarb/src/bin/scarb/args.rs:140]: So we don't need a `scarb cache-path` cli command?

File: scarb/src/bin/scarb/args.rs [hunk 4]
```diff
@@ -134,7 +136,7 @@ pub enum Command {
     Cache(CacheSubcommand),
     /// Remove generated artifacts.
     Clean,
-    /// List installed commands.
+    /// Print cache path.
```
Comment (maciektr) [inline scarb/src/bin/scarb/args.rs:139]: Can you please revert the change in this line? :D

File: scarb/src/bin/scarb/commands/cache_path.rs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+use anyhow::{Ok, Result};
+
+use scarb::core::Config;
+
+#[tracing::instrument(skip_all, level = "info")]
+pub fn run(config: &Config) -> Result<()> {
+    let path = config.dirs().cache_dir.path_unchecked();
+    println!("{path}");
```
Comment (maciektr) [inline scarb/src/bin/scarb/commands/cache_path.rs:8]: I've noticed one more thing, sorry for not mentioning this before. 

In Scarb we use our own UI abstraction for printing text to users (https://github.com/software-mansion/scarb/blob/main/scarb/src/ui/mod.rs). 
This also allows us to structure all messages to json if needed (with `--json` arg). 

Please print with `config.ui()` here (as below).

```suggestion
    config.ui().print(ValueMessage::new("path", &path));
```

**Review by maciektr — COMMENTED** [review]
Great stuff! 
Would you mind rebasing into a single commit after all changes?

**Review by maciektr — APPROVED** [review]
👏 👏 👏 
Looks great, thank you ❤️ !
